include/cadmium/engine/pdevs_dynamic_simulator.hpp

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2017, Laouen M. L. Belloli
```

Comment:
This one also looks like having a lot of copy paste from previous code, please include me :)

Reply:
Done!